tony-history-server/stopTHS.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env sh
+###########################################################################################################
+# NAME: stopTHS.sh
+#
+# DESCRIPTION:
+# This script stops the Tony History Server (THS) process.
+#
+#
+# INPUT:
+# $1 - path of RUNNING_PID of THS (Optional. Default to current folder)
+#
+#
+# EXIT CODE:
+# 0 - Success
+# 1 - Failed to find RUNNING_PID
+#
+#
+# CHANGELOG:
+# DEC 10 2018 PHAT TRAN
+############################################################################################################
+RUNNING_PID_PATH=./RUNNING_PID
```

Comment:
Currently, it seems like startTHS.sh does not create this file. Should startTHS.sh be updated?

Reply:
Synced offline. Apparently, `nohup` automatically generates a `RUNNING_PID` file.

Would be good to add a comment to startTHS.sh saying that nohup will generate the RUNNING_PID file.